bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -72,4 +77,11 @@ public Boolean isReadOnly() {
         return readOnly;
     }
 
+    @Override
+    @NotNull
+    protected ComponentData getComponentData() {
```

Comment:
Can we add this to base.

Reply:
Yes, I'll do that. It will replace the need for adding the same separately at PanelImpl and AbstractFieldImpl

Reply:
done